lockbox-ios/View/AutoLockSettingsView.swift

```diff
@@ -0,0 +1,101 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+import UIKit
+import RxSwift
+import RxCocoa
+import RxDataSources
+
+protocol AutoLockSettingsProtocol {
```

Comment:
All of the other view protocols a) have `View` in the name and b) are declared in the `Presenter`.